### PR TITLE
[KOGITO-8908] Documenting sysout arguments and log level

### PR DESCRIPTION
**Pull request by fjtirado**
**JIRA:**

https://issues.redhat.com/browse/KOGITO-8908

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc [hunk 1]
```diff
@@ -58,11 +60,7 @@ In the `state` definition, you can call the same `sysout` function as shown in t
   ]
 }
 ----
-
-[WARNING]
-====
-You must avoid using the `sysout` function in a production environment since it can spam the unnecessary data in the application log.
-====
+`message` argument can be an jq expression or a jq string using interpolation, as in the example above
```
Comment (hbelmiro) [inline serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc:63]:
```suggestion
`message` argument can be a jq expression or a jq string using interpolation, as in the example above
```

**Review by hbelmiro — CHANGES_REQUESTED** [review]

**Review by hbelmiro — APPROVED** [review]

**Comment by radtriste** [comment]
@kaldesai can you have a final look ?

**Review by MarianMacik — APPROVED** [review]

**Comment by fjtirado** [comment]
@kaldesai Can you please take a look?

File: serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc [hunk 2]
```diff
@@ -17,22 +17,24 @@ The CNCF specification does not support `java` and `sysout` functions. Therefore
...
+      "name": "logInfo",
       "type": "custom",
-      "operation": "sysout"
+      "operation": "sysout:INFO"
     }
   ]
 }
 ----
 
+The string after the : is optional and is used to indicate the log level. Possible values are TRACE, DEBUG, INFO, WARN and ERROR. If not present, INFO is assumed.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc:36]:
```suggestion
The string after the `:` is optional and is used to indicate the log level.  The possible values are TRACE, DEBUG, INFO, WARN, and ERROR. If not present, INFO is considered by default. 
```

**Review by kaldesai — APPROVED** [review]
@fjtirado, Just a small suggestion. Otherwise, it looks good to me. Thank you!

**Pull request merged**